### DISCUSSION TITLE
remote control

### DISCUSSION THREAD

**Discussion by kevinbabanats** [pybricks/support#1683] [General]
so i have a code for a GBC ball return train im working on, and the code works, but I want to use a PoweredUp remote as an override for like an e-stop incase anything happens. i cannot seem to figure out how to do that. PLEASE HELP!!!!

**Comment by laurensvalk** [comment]
You could do something like this:

![image](https://github.com/pybricks/support/assets/12326241/8fa4247e-b360-4459-adbb-caa4aeb7f897)
